src/query/stream_to_xasl.c

```diff
@@ -2336,6 +2395,29 @@ stx_build_cache_attrinfo (char *ptr)
   return ptr;
 }
 
+static char *
+stx_build_cte_xasl_id (THREAD_ENTRY * thread_p, char *ptr, XASL_ID * cte_xasl_id)
+{
+  int i;
+
+  for (i = 0; i < 5; i++)
+    {
+      cte_xasl_id->sha1.h[i] = OR_GET_INT (ptr);
+      ptr += OR_INT_SIZE;
+    }
```

Comment:
Please use the prepared macro

 OR_GET_SHA1 (ptr, &(cte_xasl_id->sha1));
 ptr += OR_SHA1_SIZE;

Reply:
OK, good point.
I fixed as your comment.